cmd/machine-config-operator/bootstrap.go

```diff
@@ -90,6 +93,7 @@ func runBootstrapCmd(cmd *cobra.Command, args []string) {
 		MachineOSContent:        bootstrapOpts.oscontentImage,
 		Etcd:                    bootstrapOpts.etcdImage,
 		SetupEtcdEnv:            bootstrapOpts.setupEtcdEnvImage,
+		EtcdQuorumGuardImage:    bootstrapOpts.etcdQuorumGuardImage,
```

Comment:
this should be hardcoded following what Colin did here https://github.com/openshift/machine-config-operator/pull/508/files#diff-8fb88a4862bafc203a34072446df1407R88 (even if it's not changing that much that way)

I believe we need to follow what's happened here as well https://github.com/openshift/machine-config-operator/pull/508

Reply:
If I'm interpreting that correctly, this is going to be two phase -- add the dummy image and set the EtcdQuorumGuardImage to the hard-coded value, land this, then back the dummy image out and change the value of EtcdQuorumGuardImage to the generic value?

Reply:
yeah (and in between, we'll land the installer PR after this, AIUI)

Reply:
Check whether I've done this correctly.

Reply:
yes, but you still need to provide a value in the cli flag (which you seem to have dropped)

Reply:
I put it in the wrong place.